MsftOpenXRGame/MsftOpenXRGame.uproject

```diff
@@ -1,6 +1,6 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "{B902521A-44AC-D40D-A44D-D684CFEBF4D2}",
+	"EngineAssociation": "4.26",
```

Comment:
This is already 4.26 in main:
https://github.com/microsoft/Microsoft-OpenXR-Unreal/blob/main/MsftOpenXRGame/MsftOpenXRGame.uproject#L3

Reply:
Oh nice, my branch was a bit stale. I've updated it.